meta/boardreports/2020-10.md

```diff
@@ -0,0 +1,68 @@
+- Activity and trusted committer diversity is still too low to sustain the project in the long run on a high activity level. Low activity evolution is sustainable.
+
+## Project Activity - Describe the overall activity in the project over the past quarter.
+
+- New process proposal by @maxcapraro, discussion (@spier @mainec @newmexicokid @lenucksi) and successful adoption (initial, structured, validated, goal: faster first contribution, see contributor handbook)
+- Processing of existing content from wiki and existing PRs into publishable repo content, large scale cleanup, migration and fixing, all ongoing for the foreseeable future. 
+- Great work towards content rendering (e.g. website, initial PDFs) in progress @spier @dizquerdo (https://innersourcecommons.gitbook.io)
+- Conference activity, more, interesting pattern activity on Slack leading to new patterns 
+- Patterns office-hours concept try out, some success, dual time-zone meeting concept successfully in use in marketing wg
+
+### Pattern-work
+
+- This is condensed overview. See [details](2020-10-details.md) for details with contributors.
+- Entirely new & merged: 5
+- Entirely new & open: 2
+- Existing PRs/Content merged: 3
+- Progress on existing PRs: 8
+- Technical contributions: ca. 8
+- Cleanup, Fixing & Migration: a lot
+- Trusted committers: 5 (active ~3)
+
+## Plans of the Project
+
+**Describe the current plans of the project. Include goals the project is working towards as well as any announcements that should be published through the marketing group.**
+
+- Publish our first official ebook of the InnerSource patterns. Once we do that, let’s announce that through our Marketing WG.
+- Process existing content from pull requests and Google group into our repository
+- Evaluate ideas to further facilitate collection of pattern content (e.g. through automation), channel ongoing discussions into pattern-work and attract more contributors, e.g. by lowering the barriers of entry for them.
+- Onboard further trusted committers
+- Possibly review the current list of trusted committers. Some of them don’t seem to be active anymore (likely they receive a lot of github/email spam from us today).
+
+## Last Committer Addition
+
+**When was the last committer added to the project? A healthy project tends to add new committers regularly. The report should indicate the most recent date on which a committer was added.**
+
+- 24th of April 2020: @lenucksi
+
+## Committer Diversity
+
+**Cover committer diversity in your project. A healthy project should survive the departure of any single contributor or employer of contributors. What are your steps to make sure that people from all regions on the globe can participate in your project without having to be awake at midnight their local time?**
```

Comment:
This is main only real objection in terms of content:
> A healthy project should survive the departure of any single contribution

This is not the case right now.

To make it explicit, if @lenucksi were to leave right now, PRs would not get merged anymore, and the feedback process would slow down to a halt. e.g for me personally as a contributor it will mean that I would likely leave the project as well because then the progress would become frustratingly slow, and I would no longer be motivated to contribute due to the lack of visible progress.

If others also see this risk, then I think we should call out this risk more explicitly. I have seen the `Activity and trusted committer diversity is still too low` above but I think we should make a stronger point about this.

This is the make or break point for the Patterns WG right now.

Reply:
The quoted sentence sets the context for the following questions. The questions have been answered.
We've added onboarding of other TCs to our plans which is what will address this.

Reply:
On this particular point my proposal is:
Let's agree to disagree :)